.dependabot/config.yml

```diff
@@ -16,3 +16,13 @@ update_configs:
                 # on false positives and breaking ai-web, so avoiding
                 # dependabot auto-update for them like other deps.
                 dependency_name: 'axe-core'
+
+    - package_manager: 'dotnet:nuget'
+      directory: '/'
+      update_schedule: 'daily'
```

Comment:
why not update_schedule: 'live'?

Reply:
we don't want this to cause noise every min :) Am thinking of reducing this to a week or something. But, thought of doing that after getting team concensus